packages/lablgtk3/lablgtk3.0.beta1/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+maintainer: "garrigue@math.nagoya-u.ac.jp"
+authors: ["Jacques Garrigue et al., Nagoya University"]
+homepage: "http://lablgtk.forge.ocamlcore.org/"
+bug-reports: "https://forge.ocamlcore.org/tracker/?group_id=220"
+dev-repo: "git+https://github.com/garrigue/lablgtk.git"
+license: "LGPL with linking exception"
+build: [
+  ["./configure" "--prefix" prefix]
+  [make "world"]
+]
+install: [
+  [make "install"]
+]
+remove: [["ocamlfind" "remove" "lablgtk3"]]
+depends: [
+  "ocaml" {>= "4.05"}
+  "ocamlfind" {>= "1.2.1"}
+]
+depopts: [
+  "conf-gtksourceview3"
+]
+depexts: [
```

Comment:
Why we didn't put this on a `conf-gtk3`?

Reply:
Because I was basing myself on the lablgtk2 opal file, written by somebody else.
I suppose it would indeed make more sense to have a separate conf-gtk3 package.
Should I do that?
By the way, the depexts are quite arbitrary.

Reply:
Having a separate conf- file is always better as it allows to have to improve/change stuff only once and not in every opam files (also it might also be handy for other packages in the future)